From 1954a7137a332bc133cac2dc745624002e2abef5 Mon Sep 17 00:00:00 2001
From: Eric Yang <eric.yang.wiwynn@gmail.com>
Date: Fri, 7 Nov 2025 08:25:45 +0000
Subject: [PATCH 15/18] pldm: Increase maximum of dbus-timeout-value

Increase the maximum value of the dbus-timeout-value option from 10 to
60.

Motivation:
In addition to handling PLDM requests, pldmd also performs D-Bus
operations in other contexts, such as MCTP endpoint discovery. If any of
these operations (e.g., getMctpEndpointProps() and other similar D-Bus
calls) time out for an existing EID, pldmd may mistakenly assume that
the MCTP EID does not exist on D-Bus and remove its MCTP info.

Increasing the maximum value of dbus-timeout-value allows greater
flexibility in configuration and helps prevent misinterpretations caused
by D-Bus timeouts.

Change-Id: I423c48592d6ddbbcab6bf3d14cc9adb126bd01fd
Signed-off-by: Eric Yang <eric.yang.wiwynn@gmail.com>
---
 meson.options | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.options b/meson.options
index d9bd868..95d1833 100644
--- a/meson.options
+++ b/meson.options
@@ -39,7 +39,7 @@ option(
     'dbus-timeout-value',
     type: 'integer',
     min: 3,
-    max: 10,
+    max: 60,
     value: 5,
     description: '''The amount of time pldm waits to get a response for a dbus
                     message before timing out''',
-- 
2.47.3

